CONTRIBUTORS.md

```diff
@@ -0,0 +1,13 @@
+## How to Contribute
+
+Please see the [read the docs](https://anemoi.readthedocs.io/en/latest/dev/contributing.html).
+
+
+## Contributors
+
+Thank you to all the wonderful people who have contributed to Anemoi. Contributions can come in many forms, including code, documentation, bug reports, feature suggestions, design, and more. A list of code-based contributors can be found [here](https://github.com/ecmwf/anemoi-core/graphs/contributors).
```

Comment:
Since we just have 1 repo and the contributors is pointing to the repo insights, do we keep 4 contributors files?

Reply:
I'd say yes. Just for consistency and it's easy now, since we have a singular file that gets broadcast.